Makefile

```diff
@@ -328,7 +328,7 @@ messaging: node-modules types $(shell find $(messaging)/src $(find_options))
 	$(docker_run) "cd modules/messaging && npm run build"
 	$(log_finish) && mv -f $(totalTime) $(flags)/$@
 
-node: cf-core contracts types apps messaging $(shell find $(node)/src $(node)/migrations $(find_options))
+node: cf-core types contracts crypto apps messaging $(shell find $(node)/src $(node)/migrations $(find_options))
```

Comment:
if the `node` rule needs the `crypto` dep, then shouldn't the client? and since its a `cf-core` dep, doesn't that mean that including `cf-core` will force the `crypto` module to get built anyways?

Reply:
definitely we should also cleanup duplicated `types` build commands